.gitignore

```diff
@@ -158,3 +158,5 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
+
+local/
```

Comment:
Include a 'local/' directory to store files locally that we don't want to upload to the repository. such as example code, notes, passwords, etc.